docs/cities/manchester/teaching-resources.md

```diff
@@ -0,0 +1,39 @@
+---
+id: teaching-resources
+title: Teaching Resources
+---
+
+> **Note:**
+>
+> Resoures here is quite new and recent. We are trying different ways to consolate our collective knowledge, it will be subjected to change.
+
+Teaching resources is used as a single source of truth that all members of CYF can access.
+
+## HTML & CSS
+
+### Week 1
+
+#### Lesson Information
+
+- [Lesson Week 1 [GitHub]](https://github.com/anthonytranDev/cyf-html-css/blob/master/week-1/LESSON.md)
```

Comment:
This information is duplicated further down as a part of the Week 1 syllabus page. Suggest not splitting parts of the syllabus page out, as it removes it from content and menu structure.

Reply:
We are assuming students will read and go through the entire week-1 directory, so much so that they would come across Lesson.md files and actually read it.

Lesson.md is one of the most important documents in the directory, and I believe it's important to prioritise this.

Please let me know what you think - up for discussion